### ISSUE TITLE
Multi Language

### ISSUE THREAD

**Issue by yaniv-peretz** [Midburn/spark#659]
Hi @alechko,
As part on the long term vision I assume we are trying to enable multiple language support,
in order to do so I propose to add a LANGUAGES const into .env.
I've added to .env-defualts, if agreed, we should ask ORI to update I assume?

reason is that by doing so, will be able to run tools, programmability.
example: for the test scripts, if we have the LANGUAGES setup, we can then loop the test on the page for each supported language.
also, we could treat the LANGUAGES[0] as main language, example: currently if the \:lng\ is omitted on some pages, then spark will automatically add \he\ I suppose in the feature users from other countries would like different defaults.

**Comment by amotenko** [comment]
If I understand correctly you suggest adding an environment variable that indicates the current language for testing purposes?

I'm @amotenko btw, @alechko is someone else :)

**Comment by yaniv-peretz** [comment]
Lol :) sorry.

Yes, but not only for testing, but testing is a use case. another example: 
        middleware.addRoute(i18next, '/:lng', ['en', 'he'], app, 'get', function (req, res) {
(app.js 135) - currently hard coding 'en' & 'he' . should use .env instead to allow other languages per requirement.

assuming we/some1 would like to add Russian lng support, he/she will need to change many hard coded variables.
there are many more examples in the system, just look for 'he' or 'en' , most of the should be changed to a .env.langague[0] or so.

is it clear now?

**Comment by amotenko** [comment]
yes. You are saying that the knowledge of which languages are supported shouldn't be hard-coded and languages should be referenced by a serial number, not by name.
I agree.

**Comment by yaniv-peretz** [comment]
cool,
I've added
```
#supported Languges
LANGUAGES=["en","he"]
```

into dotenv, but we need to update the .env for production & stagging.
@OriHoch maybe could do it? (I'm not sure how to reach it on the severs)
